app/routers/profile.py

```diff
@@ -194,3 +195,39 @@ async def update(
     finally:
         url = router.url_path_for("profile")
         return RedirectResponse(url=url, status_code=HTTP_302_FOUND)
+
+
+@router.get("/restore_events")
+async def restore_events(request: Request,
+                         session=Depends(get_db)):
+    # TODO: Add user.id instead 1
+    deleted_events = (session.query(Event.id,
+                                    Event.title,
+                                    Event.start,
+                                    Event.end).
+                      filter(Event.owner_id == 1,
+                             Event.deleted_date.isnot(None)).all())
+
+    return templates.TemplateResponse("restore_events.html", {
+        "request": request,
+        "deleted_events": deleted_events
+    })
+
+
+@router.post("/restore_events")
+async def restore_events_post(request: Request,
+                              session=Depends(get_db)):
+    data = await request.form()
+    events_ids_to_restored = get_events_ids_to_restored(data._list)
+
+    # TODO: Add user.id instead 1
+    restore_del_events = (session.query(Event).
+                          filter(Event.owner_id == 1,
```

Comment:
Please write the owner.id as user.id and leave it in the comment

Reply:
Thanks , I changed